### PR TITLE
Html css

### PR DESCRIPTION
**Pull request by keneogu**
Tested me github and also learned how to use it fluently.
Added a html file
Added css file
Added .gitignore file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# Hello-World
\ No newline at end of file
+# Creating Github Repository
+##testing my github repo
```
Comment (tumwinep) [inline README.md]: Add Space after the second hash

**Review by tumwinep — APPROVED** [review]
Please find comments below

File: README.md (hunk 1 above)
Comment (keneogu) [reply README.md]: have done that

**Review by keneogu — COMMENTED** [review]
thank you for your review

**Comment by keneogu** [comment]
thank you for your approval

**Pull request merged**